wgs/cohort_qc.py

```diff
@@ -11,17 +11,18 @@ def cohort_qc_workflow(args):
 
     workflow = pypeliner.workflow.Workflow(ctx=helpers.get_default_ctx(docker_image=config.containers('wgs')))
 
-    inputs = helpers.load_yaml(args['input_yaml'])
+    inputs = helpers.lod_qc_input_yaml_flat(args['input_yaml'])
     out_dir = args["out_dir"]
     tmp_dir = args["tmpdir"]
     api_key = args["API_key"]
 
-    cohort_mafs = {cohort_label: data["cohort_maf"] for cohort_label, data in inputs.items()}
+    sample_mafs = {label: data["sample_maf"] for label, data in inputs.items()}
+    sample_labels = {label: data["sample_label"] for label, data in inputs.items()}
```

Comment:
what's in sample_label??

Reply:
so this is something we may want to think about how to address. Andrew prefers these cohort analyses to be done on a patient, not sample level. In the previous iteration of this pipeline, we were taking in a pre-merged maf, so it was up to the user to do the legwork of creating sample labels in the data. However, with sample-level mafs as input, it makes more sense i think to do patient labeling by default in the pipeline. so sample_label is a patient group, or some meta-labeling, that the user wants in the analysis. I think its nicely flexible that way, because if the user wants to maintain sample-groups, they can use them for sample_label